cishouseholds/impute.py

```diff
@@ -359,10 +337,15 @@ def fill_forward_event(
     for col in event_columns:
         df = df.withColumn(col, F.when(F.col("DROP_EVENT"), None).otherwise(F.col(col)))
 
+    # pick the best row to retain based upon proximity to visit date
```

Comment:
We no longer need to prioritise responses based on difference in days between event_date and visit_date.  https://jira.ons.gov.uk/browse/CISDP-2795 

We simply need to prioritise the original response within the tolerance window

Reply:
Stef ran through this yesterday, due to the current implementation we have a long event dataframe that we are disambiguating based on visit_diff rather than visit_datetime so that when we are joining the event dataframe back on visit_datetime it prefers events that are closest to the visit_datetime. Without this it would end up joining the first event onto each visit_datetime.